internal/plugin/grpc_provider.go

```diff
@@ -27,6 +25,16 @@ import (
 
 var logger = logging.HCLogger()
 
+// Some providers may generate quite large schemas, and the internal default
+// grpc response size limit is 4MB. 64MB should cover most any use case, and
+// if we get providers nearing that we may want to consider a finer-grained
+// API to fetch individual resource schemas.
+// Note: this option is marked as EXPERIMENTAL in the grpc API. We keep
+// this for compatibility, but recent providers all set the max message
+// size much higher on the server side, which is the supported method for
+// determining payload size.
```

Comment:
Factoring out the constant does seem reasonable, but probably we should keep this part of the comment discussing "this option" being marked as experimental close to the `grpc.MaxRecvMsgSizeCallOption` reference, since that is what "this option" is referring to, rather than to the specific value being passed to it.